packages/settingregistry/src/tokens.ts

```diff
@@ -27,6 +35,12 @@ export const ISettingRegistry = new Token<ISettingRegistry>(
   See "schemaDir" for more information.`
 );
 
+/**
+ * The settings connector interface.
+ */
+export interface ISettingsConnector
```

Comment:
Should it be singular or plural to match the name of the class?

Reply:
Good catch, thanks!